Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
     "near-abi-client",
+    "near-abi-client-impl",
     "near-abi-client-macros",
 ]
-exclude = ["examples/"]
\ No newline at end of file
+exclude = ["examples/"]
```

Comment:
@itegulov is there a specific reason we're excluding the examples from the workspace? would be nice to reuse the same target directory across this workspace and run all the tests in the workspace with a simple `cargo test`.

Reply:
I am just following `near-sdk-rs` so that all examples are as close to real-world projects as they can be. Not sure if that matters all that much; feel free to put them under the root workspace. I find that it is pretty annoying to have to open multiple VS Code instances to work on examples, so I would welcome this change.

Reply:
#6